requirements.prereq.txt

```diff
@@ -0,0 +1,5 @@
+six==1.10.0
```

Comment:
the Dockerfile does reference https://github.com/elifesciences/sciencebeam-gym/blob/9d3fa6d081d15a543463bb85d7fcce768f4e0c6b/Dockerfile#L8 a typo in the new filename?

Reply:
Thanks... going to close this out now as fixing it by using the latest version of `tensorflow-transform`